internal/common/filespec_test.go

```diff
@@ -146,6 +146,81 @@ func getCopyTestCases() []*testCaseCopy {
 			true,
 			"720b5768bf364f35c316976e549f1bcd",
 		},
+		&testCaseCopy{
+			FileSpec{Type: "path", Path: filepath.Join(config.GetTestdataDir(), "files", "*_test"), Dest: "dest"},
+			true,
+			"87efd29f124f9a5b617517041d4d9b32",
```

Comment:
These tests are starting to get complex enough that I think we need another way to handle the expected output to make it more readable.
Think through ways you may want to do it.
It's a big hard because there can be multiple files and dirs.

Maybe we assume that the test dir is the base (relative dir), so we no longer have to supply it.
Then the expected could be a `map[string]any` (or JSON) that describes what the result should look like.
Dirs could be keys for their contents, and files could be keys for some constant that represents each file.
```
# Single file
map[string]any{
    "test.txt": SPEC_TXT,
}

# Directory with file.
map[string]any{
    "filespec_test": {
        "test.txt": SPEC_TXT,
    },
}
```

There's several ways you could make the expected output more clear,
the above is just one suggestion.